prover/src/constraints/evaluator.rs

```diff
@@ -264,9 +269,9 @@ impl<'a, A: Air, E: FieldElement<BaseField = A::BaseField>> ConstraintEvaluator<
     fn evaluate_main_transition(
         &self,
         main_frame: &EvaluationFrame<E::BaseField>,
-        x: E::BaseField,
         step: usize,
         evaluations: &mut [E::BaseField],
+        domain: &StarkDomain<A::BaseField>,
```

Comment:
minor nit: I'd probably put `domain` parameter right before `step` parameter here and below as well.

Reply:
Done!